src/main/proto/addressbook.proto

```diff
@@ -0,0 +1,30 @@
+syntax = "proto2";
```

Comment:
I believe the current version is "proto3", the one used by grpc. It could be interesting to let readers know that messages could interop between the 2?

Reply:
I will change that, I took the example from the Protocol Buffers website.